subroutines.sh

```diff
@@ -208,6 +208,18 @@ send_key() {
   done
 }
 
+scan_bleln_macaddr(){
+  # copied from legacy "scan_bluetooth" function. To decide if still relevant
+  # note there is this PR https://github.com/tesla-local-control/tesla-local-control-addon/pull/32
+  # quite old, but has the principles for auto populating the BLE MAC Address with only the VIN
+  vin=$1
+  VIN_HASH=$(echo -n ${vin} | sha1sum)
```

Comment:
We have a function for this... `BLE_ADVERT=$(tesla_vin2ble_ln $vin)`

Reply:
oh ok I see. I will update